docker-compose.yml

```diff
@@ -16,11 +17,12 @@ services:
     volumes:
       - './configs/peer:/config'
     init: true
-    command: ./iroha --submit-genesis
+    command: iroha --submit-genesis
 
   iroha1:
     image: hyperledger/iroha2:dev
     environment:
+      KURA_BLOCK_STORE_PATH: /storage
```

Comment:
According to `docker-compose-local.yml`, perhaps needs to add `depends_on: - iroha0` here that peers who submit genesis will depend on a peer who submits genesis.

Reply:
Since in the real production the genesis peer should start without knowing the network status, I don't think the order of service start should be specified in general. The reason why it is specified in `docker-compose-local.yml` is for building the image, I think

Reply:
However, we don't have `depends_on:` block in docker-compose template for iroha2 deployment on our real stands (environments).